conda/activate.py

```diff
@@ -65,8 +63,7 @@ def __init__(self, arguments=None):
         self._raw_arguments = arguments
         self.environ = os.environ.copy()
 
-    # Once Python2 dies odargs can become kwargs again since dicts are ordered since 3.6.
-    def get_export_unset_vars(self, odargs):
+    def get_export_unset_vars(self, *, conda_exe_vars=True, **kwargs):
```

Comment:
Maybe it would make more sense to have `conda_exe_vars` be `export_conda_vars` instead? I say this because associating a boolean with a verb (or action in this case) is perhaps a little clearer. When I see a plural noun like `conda_exe_vars`, I instantly think, "oh this is going to be a sequence or something".

Just a suggestion, no need to implement